src/test/java/tests/yandex_email/BaseStep.java

```diff
@@ -0,0 +1,34 @@
+package tests.yandex_email;
+
+import com.google.common.collect.ImmutableMap;
+import driver.Config;
+import driver.DriverSingleton;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.openqa.selenium.WebDriver;
+
+import static com.github.automatedowl.tools.AllureEnvironmentWriter.allureEnvironmentWriter;
+
+public class BaseStep {
+    static WebDriver driver;
```

Comment:
bad idea to make this static, as methods below. Static is a very very bad word, and it brings a lot of issue in multithreading env. So, please read about static and thread, and remove it from here./

Reply:
fixed